### DISCUSSION TITLE
How does authentication work?

### DISCUSSION THREAD

**Discussion by BernardRat** [Tyrrrz/YoutubeDownloader#507] [Q&A]
When I click on the authentication icon, I just get a black window with Authentication at the top and Close at the bottom,  without any prompts.  The  I'm on latest version using Win 10 and everything is up to date.  Otherwise, the downloader works great.  

It would also be nice if there was a button to clear the URL text so I can add another URL text without having to ctrlA and delete first.

Thanks

**Comment by Tyrrrz** [comment]
If you get a black window, you need to manually install WebView2 runtime: https://developer.microsoft.com/en-us/microsoft-edge/webview2

**Reply by BernardRat** [reply]
Thanks Tyrrrz, that was the problem.  All OK now.
